src/shared/prompts.js

```diff
@@ -0,0 +1,59 @@
+import { render } from 'preact';
```

Comment:
Do you have a sense of where you'd see this module residing in the `frontend-shared` package later?

Reply:
Hmm... I'm not sure yet. I think we can just re-exports its functions directly from the root so clients can do `import { confirm } from '@hypothesis/frontend-shared'`.